specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-01-01/frontdoor.json

```diff
@@ -0,0 +1,2292 @@
+        {
+          "$ref": "#/definitions/RouteConfiguration"
+        }
+      ]
+    },
+    "RedirectConfiguration": {
+      "description": "Describes Redirect Route.",
+      "x-ms-discriminator-value": "#Microsoft.Azure.FrontDoor.Models.FrontdoorRedirectConfiguration",
+      "properties": {
+        "redirectType": {
+          "description": "The redirect type the rule will use when redirecting traffic.",
+          "type": "string",
+          "enum": [
+            "Moved",
+            "Found",
+            "TemporaryRedirect",
+            "PermanentRedirect"
+          ],
+          "x-ms-enum": {
+            "name": "FrontDoorRedirectType",
+            "modelAsString": true
+          }
+        },
+        "redirectProtocol": {
+          "description": "The protocol of the destination to where the traffic is redirected",
+          "type": "string",
+          "enum": [
+            "HttpOnly",
+            "HttpsOnly",
+            "MatchRequest"
+          ],
+          "x-ms-enum": {
+            "name": "FrontDoorRedirectProtocol",
+            "modelAsString": true
+          }
+        },
+        "customHost": {
+          "type": "string",
+          "description": "Host to redirect. Leave empty to use the incoming host as the destination host."
+        },
+        "customPath": {
+          "type": "string",
+          "description": "The full path to redirect. Path cannot be empty and must start with /. Leave empty to use the incoming path as destination path."
+        },
+        "customFragment": {
+          "type": "string",
+          "description": "Fragment to add to the redirect URL. Fragment is the part of the URL that comes after #. Do not include the #."
+        },
+        "customQueryString": {
+          "description": "The set of query strings to be placed in the redirect URL. Setting this value would replace any existing query string; leave empty to preserve the incoming query string. Query string must be in <key>=<value> format. The first ? and & will be added automatically so do not include them in the front, but do separate multiple query strings with &.",
+          "type": "string"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/RouteConfiguration"
+        }
+      ]
+    },
+    "Backend": {
+      "description": "Backend address of a frontDoor load balancer.",
+      "type": "object",
+      "properties": {
+        "address": {
+          "description": "Location of the backend (IP address or FQDN)",
+          "type": "string"
+        },
+        "privateLinkId" : {
+          "description": "If this backend is private, provide the private link resource Id. Populating this optional field indicates that this backend is 'Private'",
+          "type": "string"
+        },
+        "location": {
+          "description": "Region of the backend if it is in Azure. Mandatory field if backend is 'Private'",
+          "type": "string"
+        },
```

Comment:
same here.. breaking change guide : [https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo](https://microsoft.sharepoint.com/:w:/t/azureresourcemanagerteam/EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw?e=xEcppo)

New property added to response 

If a new property/field is added to the response an API, the GET-PUT pipeline will be broken. Consider the case where from portal a customer updates the value of a new property "A". Another customer does a GET of this resource using the SDK. The SDK will ignore the property since it does not understand it. From the SDK, the customer does a PUT using the model that was returned from the GET. This will overwrite the change made by the first customer from the portal.

Reply:
Same as above